packages/core/test-helpers/core-test-helpers-kbn-server/src/create_serverless_root.ts

```diff
@@ -113,22 +113,44 @@ const getServerlessESClient = () => {
   });
 };
 
-const defaults = {
-  server: {
-    restrictInternalApis: true,
-    versioned: {
-      versionResolution: 'newest',
-      strictClientVersionCheck: false,
+const getServerlessDefault = () => {
+  return {
+    server: {
+      restrictInternalApis: true,
+      versioned: {
+        versionResolution: 'newest',
+        strictClientVersionCheck: false,
+      },
+    },
+    migrations: {
+      algorithm: 'zdt',
+      zdt: {
+        runOnRoles: ['ui'],
+      },
```

Comment:
Not needed for this PR, but I thought we might as well ensure these tests keep running migrations. Open to other approaches here.

Reply:
👍 

I'm thinking about https://github.com/elastic/kibana/pull/162673 though, and we should probably also add `runOnRoles: ['ui']` on the base FTR config(s) for serverless FTR suite?